frontend/components/organiser/event/attendee/tabs/ApprovedAttendeeTab.tsx

```
@@ -0,0 +1,258 @@
+    );
+  };
+  return (
+    <>
+      <div className="flex justify-between">
+        <div className="text-2xl md:text-4xl font-extrabold">Approved Attendees</div>
+        <div className="flex items-center space-x-4">
+          <DownloadCsvButton data={allAttendeesCsvData} headers={csvHeaders} filename={`Attendees_${eventId}.csv`} />
+          <div
+            className="inline-flex justify-center rounded-md bg-organiser-dark-gray-text px-2 md:px-4 py-1.5 md:py-2 text-sm font-medium text-white hover:bg-black/30 focus:outline-none focus-visible:ring-2 focus-visible:ring-white/75 hover:cursor-pointer"
+            onClick={() => setIsFilterModalOpen(true)}
+          >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)